ssha/ssm.py

```diff
@@ -81,6 +87,41 @@ def find_instances():
     return instance_info_list
 
 
+def generate_key():
+    path = os.environ['HOME'] + "/.ssh/ssha/"
```

Comment:
I think this function would be better in a few ways if it used tempfiles instead of managing a `/.ssh/ssha/` directory.